plugin-api/build.gradle

```diff
@@ -56,7 +61,7 @@ Calculated : ${currentHash}
 tasks.register('checkAPIChanges', FileStateChecker) {
   description = "Checks that the API for the Plugin-API project does not change without deliberate thought"
   files = sourceSets.main.allJava.files
-  knownHash = 'uU7ZwzC+EVf5o1eF+gxYgxLLOhAvKq7uLCKw3JSldjY='
+  knownHash = 'NpgLMMVnEhos2Z1ze7A3WNvaWs+8u8Cf3lTBWwPR7xY='
```

Comment:
All of the changes in the plugins-api need to be rolled back.  We have other plugins already that depend on these signatures, we cannot change any of the types returned or passed in.  I don't see a reason we need to do any changes in this PR that would cause this signature hash to change.

Reply:
done.